tests/automation_framework/src/work_order_submit/work_order_submit_params.py

```diff
@@ -67,7 +67,7 @@ def add_json_values(self, input_json_temp, pre_test_response,
                     value = value.encode("UTF-8").hex()
                 wconfig.set_parameter(self.params_obj, c_key, value)
 
-        self.params_obj["workerEncryptionKey"] = self.params_obj["workerEncryptionKey"].encode("UTF-8").hex()
+        self.params_obj["workerEncryptionKey"] = self.params_obj.get("workerEncryptionKey", '').encode("UTF-8").hex()
```

Comment:
what is this change?

Reply:
Similar Change. In case workerEncryptionKey does not exist